sanic/app.py

```diff
@@ -451,7 +451,13 @@ def add_route(
 
     # Decorator
     def websocket(
-        self, uri, host=None, strict_slashes=None, subprotocols=None, name=None
+        self,
+        uri,
+        host=None,
+        strict_slashes=None,
+        subprotocols=None,
+        version=None,
```

Comment:
I believe we should add the `version` parameter after the already existing `name` parameter since some people may be using positional args instead of named args and we don't want to break those users

Reply:
I did it this way to be consistent with the order of the parameters on the other routes where name is the final.

What's worse? Inconsistent ordering or changing positional order?

Reply:
Maybe nobody really used that feature before. Nobody has complained about that bug. 😅